src/lambda/scrape.js

```diff
@@ -89,9 +81,11 @@ exports.handler = async (event, context) => {
             body: JSON.stringify(err)
         };
     }
+    
+    const body = { lyricsData };
 
     return {
         statusCode: 200,
-        body: JSON.stringify(lyricsData)
+        body: JSON.stringify(body)
```

Comment:
How come you're nesting the data one more object deep? Rather than just returning lyricsData by itself?

Reply:
Well initially it was just because I realised that Results.js was trying to access a 'lyricsData' property on the returned object, so I just fixed it on this end rather on the other - eventually it might make sense to go back to this way if we're going to return translated/untranslated lyrics, but I'll change it now in the interest of simplicity!